src/Elcodi/Component/Banner/Services/BannerManager.php

```diff
@@ -57,7 +57,7 @@ public function getBannersFromBannerZoneCode($bannerZoneCode, LanguageInterface
     {
         return $this
             ->bannerRepository
-            ->getBannerByZone($bannerZoneCode, $language);
+            ->getBannerByZone((string) $bannerZoneCode, $language);
```

Comment:
Hmmm, be careful here. This casting is pretty interesting if it is done only at the model.
What do you think?